articles/advanced-r-concepts/index.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/advanced-r-concepts/
+title: Advanced Programming Concepts in R
+description:
+author: lalithnarayan-c
+date: 2020-10-15T00:00:00-11:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/advanced-r-concepts/hero.jpg
+    alt:
+---
+In the first article, we have covered the basics of the R programming language. In this article, we will look at some of the advanced aspects of the R programming language. Going through this article will enhance one's ability to implement Object-Oriented Programming(OOP) concepts and write modular code. We will cover the following topics in this article:
```

Comment:
"we have covered" => "we covered" (use simple present tense rather than present perfect when possible

space between Programming and (OOP)